payload-schemas/index.js

```diff
@@ -0,0 +1,69 @@
+const Ajv = require('ajv');
```

Comment:
Could it be possible you forgot to commit this devDependency in `package.json` + `package-lock.json`?

Reply:
Oops, definitely so. Thanks!

Reply:
Oh! It was already in the project! Didn't notice that. Sorry 🙏🏽